src/libraries/System.IO.FileSystem.AccessControl/src/System.IO.FileSystem.AccessControl.csproj

```diff
@@ -46,6 +46,8 @@
              Link="Common\Interop\Windows\Interop.GET_FILEEX_INFO_LEVELS.cs" />
     <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.GetFileAttributesEx.cs"
              Link="Common\Interop\Windows\Interop.GetFileAttributesEx.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.GetFileInformationByHandle.cs"
+             Link="Common\Interop\Windows\Interop.GetFileInformationByHandle.cs" />
```

Comment:
Did you find out why is this needed? I can investigate a bit more but I will have to build your PR locally.

Reply:
Its because `WIN32_FILE_ATTRIBUTE_DATA.cs` requires these two files